numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -3108,7 +3108,9 @@ hpy_type_tuple_type_resolver(HPyContext *ctx,
 
     ufunc_name = ufunc_get_name_cstr(self_struct);
 
-    use_min_scalar = should_use_min_scalar(nin, op, 0, NULL);
+    hpy_abort_not_implemented("convert op to PyArrayObject**");
+    PyArrayObject ** dummy;
+    use_min_scalar = should_use_min_scalar(nin, dummy, 0, NULL);
```

Comment:
should_use_min_scalar is a legacy interface, `op` is a `HPy *`

Reply:
Is there something preventing us from calling `HPy_AsPyObject(op)` here:?

Reply:
That might work, but I would prefer not to add more `HPy_AsPyObject`.